### PR TITLE
puppetdb_exporter: Update 1.0.0->1.1.0

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

**Comment by bastelfreak** [comment]
we should wait for 1.1.1 https://github.com/camptocamp/prometheus-puppetdb-exporter/issues/9

**Comment by bastelfreak** [comment]
upstream fixed it :+1:

File: REFERENCE.md [hunk 1]
```diff
@@ -45,6 +45,7 @@ The package method needs specific yum or apt repo settings which are not made ye
 * [`prometheus::server`](#prometheusserver): class to manage the actual prometheus server. This class gets called from the init.pp
 * [`prometheus::snmp_exporter`](#prometheussnmp_exporter): This module manages prometheus snmp_exporter
 * [`prometheus::statsd_exporter`](#prometheusstatsd_exporter): This module manages prometheus statsd_exporter
+* [`prometheus::unbound_exporter`](#prometheusunbound_exporter): This module manages prometheus node node_exporter
```
Comment (alexjfisher) [inline REFERENCE.md]: Is this supposed to be in this PR?

File: manifests/puppetdb_exporter.pp [hunk 2]
```diff
@@ -55,7 +55,7 @@
   String[1] $package_ensure               = 'present',
   String[1] $package_name                 = 'puppetdb_exporter',
   String[1] $user                         = 'puppetdb-exporter',
-  String[1] $version                      = '1.0.0',
+  String[1] $version                      = '1.1.0',
```
Comment (alexjfisher) [inline manifests/puppetdb_exporter.pp:58]: I think changing the default version installed is a breaking change?  Perhaps a several default exporter versions could be batched into a single release?

File: REFERENCE.md (hunk 1 above)
Comment (bastelfreak) [reply REFERENCE.md]: ah. that sneaked in while I was working on the other PR

Comment (bastelfreak) [reply REFERENCE.md]: fixed it

File: manifests/puppetdb_exporter.pp (hunk 2 above)
Comment (bastelfreak) [reply manifests/puppetdb_exporter.pp:58]: yes breaking change. But I would like to update the other exporters on individual PRs (and we already have a few other backwards-incompatible changes on master)

**Review by dhollinger — APPROVED** [review]
LGTM

**Pull request merged**